### PR TITLE
Migrated to github.com/getsentry/sentry-go.

### PR DESCRIPTION
**Pull request by lunemec**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,13 @@
+module github.com/lunemec/zapext
```
Comment (tchap) [inline go.mod]: Not sure this is the right module name 🙂

File: go.mod [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+go 1.13
+
+require (
+	github.com/certifi/gocertifi v0.0.0-20200211180108-c7c1fbc02894 // indirect
+	github.com/getsentry/raven-go v0.2.0
```
Comment (tchap) [inline go.mod]: Why do we need both?

File: zapsentry/example/main.go [hunk 3]
```diff
@@ -5,9 +5,9 @@ import (
 	"fmt"
 	"os"
 
+	"github.com/getsentry/raven-go"
```
Comment (tchap) [inline zapsentry/example/main.go]: I am not sure what this is about...

**Comment by tchap** [comment]
Thanks for the PR @lunemec . I will check this later on. BTW does this change the API? Shall we release as 2.0? I guess so since some fields are processed differently. And also, would be good to just go mod init as a separate PR so that it does not pollute this... Or at least a separate commit.

File: go.mod (hunk 1 above)
Comment (lunemec) [reply go.mod]: Yeah. I hate Go modules. WTF is this shit. Ideas what to do about this except to not commit this?

File: go.mod (hunk 2 above)
Comment (lunemec) [reply go.mod]: I dunno. Ran `go mod tidy` but I expect it remained in the test or example.

File: zapsentry/example/main.go (hunk 3 above)
Comment (lunemec) [reply zapsentry/example/main.go]: Yeah, I did not look at that. Sorry.

**Comment by lunemec** [comment]
@tchap yeah, 2.0. It removes some public methods for Options that do not make sense anymore.

File: go.mod (hunk 1 above)
Comment (tchap) [reply go.mod]: Sure, just use `github.com/tchap/zapext`. This does not mean anything when you are developing the package. You can just clone it from your fork. But if you want to use it under `github.com/tchap/zapext` in your projects for testing, I guess you need to use [replace](https://github.com/golang/go/wiki/Modules#when-should-i-use-the-replace-directive)?

File: zapsentry/example/main.go (hunk 3 above)
Comment (tchap) [reply zapsentry/example/main.go]: I guess this is the reason the dep on `raven-go` stayed in `go.mod`...

**Comment by tchap** [comment]
> @tchap yeah, 2.0. It removes some public methods for Options that do not make sense anymore.

So I guess there are two ways. We either create `v2.0.0` release or we place the new package into `/v2` directory as far as I know. Which one do we want to do?

**Comment by lunemec** [comment]
`v2.0.0` release, since the old library (raven-go) is deprecated.

**Review by tchap — APPROVED** [review]

**Comment by tchap** [comment]
**LGTM** 👍 

Can you squash please before I merge stuff?

**Comment by lunemec** [comment]
@tchap done

**Pull request merged**